apps/judicial-system/backend/src/app/formatters/casefilesPdf.ts

```diff
@@ -28,18 +33,18 @@ function constructCasefilesPdf(
 
   doc
     .font('Helvetica-Bold')
-    .fontSize(26)
+    .fontSize(hugeFontSize)
     .lineGap(8)
     .text('Rannsóknargögn', { align: 'center' })
     .font('Helvetica')
-    .fontSize(18)
+    .fontSize(largeFontSize)
     .lineGap(40)
     .text(
       `Mál nr. ${existingCase.courtCaseNumber} - LÖKE nr. ${existingCase.policeCaseNumber}`,
       { align: 'center' },
     )
     .lineGap(8)
-    .fontSize(12)
+    .fontSize(baseFontSize)
```

Comment:
This will change from font size 12 to font size 11.
Is that what we want here?

Reply:
I asked this question in the Asana task, waiting for an answer.. Will report back